meta_configurator/src/components/gui-editor/properties/OneOfSelectionProperty.vue

```diff
@@ -19,8 +19,17 @@ const props = defineProps<{
   propertyData: any | undefined;
   absolutePath: Path;
   possibleSchemas: Array<JsonSchema>;
+  isTypeUnion: boolean;
```

Comment:
this is the hacky part: a boolean that changes the logic of this component. 
I haven't yet come up with a nice way to implement this better, but probably it would involve refactoring this component a lot 
Just passing the map of the current selected options doesn't work for some reason, I think it has to do with the reactivity

Reply:
I will just create an issue to refactor and test the AnyOf and OneOf Selection Properties